src/js/components/Marathon.jsx

```diff
@@ -178,6 +181,15 @@ var Marathon = React.createClass({
     );
   },
 
+  getChaosModal: function () {
+    return (
+      <ChaosModalComponent
+        isChaosInProgress={this.state.isChaosInProgress}
```

Comment:
So you suggest to skip this check entirely? And let the user submit as many Chaos(es?) as he sees fit?